### DISCUSSION TITLE
PowerShell App Deployment Toolkit 3.9.1

### DISCUSSION THREAD

**Discussion by seanlillis** [PSAppDeployToolkit/PSAppDeployToolkit#738] [General]
Version 3.9.1 [20/01/2023]
- Resolved an issue introduced in v3.9.0 where toast notifications would not display when run in the system context (#733)
- Added Finnish language translations to the UI components (#681)
- Added variable $IsMultiSessionOS for virtual environments (#697)
- Add missing dash before TypeName parameter used with New-Object in the in Set-ItemPermission function which resulted in failing to set read permissions on the BlockExecution path. (#721)
- Resolved an issue introduced in v3.8.4 where registry entries set by Block-AppExecution would not be cleaned up after an interrupted installation (#714)
- Resolved an issue where a PowerShell prompt would flash on screen when invoking a PowerShell script using Execute-ProcessAsUser (#735)
- Added the appropriate exit code 1638 for MSIs that are already installed when using the Execute-MSI function with -PassThru (#736)
- Added creation of the $dirAppDeployTemp directory by default so the variable can be used for general actions (#708)

## What's Changed
* Execute-ProcessAsUser VBScript that will start PowerShell fix by @bshiznit in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/pull/735
* Get-InstalledApplication Doku update by @Akitarmo in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/pull/669
* Fixes the error returned by the scheduled task created by Block-AppExecution by @denisrennesOR in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/pull/715

## New Contributors
* @bshiznit made their first contribution in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/pull/735
* @denisrennesOR made their first contribution in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/pull/715

**Full Changelog**: https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/compare/3.9.0...3.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/releases/tag/3.9.1'>PowerShell App Deployment Toolkit 3.9.1</a>.</em>

**Comment by MGSchoe** [comment]
Hello, is there an option to disable the ExecuteAsUser function run at the end of the script? We're using proces-whitelisting at our branch and starting the powershell proces under user credentials is blocked when running the script under system account from SCCM

**Reply by seanlillis** [reply]
I've pushed a fix for this to the master branch (#741)

**Comment by ajscott** [comment]
The new Finnish portion of the XML along with a few of the sections in AppDeployToolkitMain.ps1 still use tabs instead of spaces for indents.

**Reply by seanlillis** [reply]
Will be fixed in the next release. Thanks.
